lib/web_ui/test/engine/semantics/semantics_helper_test.dart

```diff
@@ -122,14 +119,7 @@ void testMain() {
 
       test('Non-relevant events should be forwarded to the framework',
           () async {
-        DomEvent event;
-        if (_defaultSupportDetector.hasPointerEvents) {
-          event = createDomPointerEvent('pointermove');
-        } else if (_defaultSupportDetector.hasTouchEvents) {
-          event = createDomTouchEvent('touchcancel');
```

Comment:
`createDomTouchEvent` and `createDomMouseEvent` are now probably unused in `dom.dart`?

Reply:
`createDomTouchEvent` yes, but `createDomMouseEvent` is used in other places.